runtime-api/bonded-currency/Cargo.toml

```
@@ -0,0 +1,33 @@
+[package]
+authors       = { workspace = true }
+description   = "Runtime APIs for dealing with the bonded currency pallet."
```

Let's stick to the same naming everywhere?
```suggestion
description   = "Runtime APIs for dealing with the bonded coins pallet."
```

---

[2ac860e](https://github.com/KILTprotocol/kilt-node/pull/828/commits/2ac860e758449aeba7ab6de34819ce941f89d9b0)